package.json

```diff
@@ -24,14 +36,26 @@
   },
   "homepage": "https://github.com/ethereumjs/ethereumjs-account#readme",
   "dependencies": {
-    "ethereumjs-util": "^5.0.0",
-    "rlp": "^2.0.0",
+    "bn.js": "^4.11.8",
+    "ethereumjs-util": "^6.0.0",
+    "rlp": "^2.2.1",
     "safe-buffer": "^5.1.1"
   },
   "devDependencies": {
+    "@ethereumjs/config-nyc": "^1.0.0",
+    "@ethereumjs/config-prettier": "^1.0.1",
+    "@ethereumjs/config-tsc": "^1.0.2",
+    "@ethereumjs/config-tslint": "^1.0.0",
+    "@types/bn.js": "^4.11.3",
```

Comment:
Same for the types?

Reply:
I'm not sure why, but without this typescript fails with:

```
node_modules/rlp/dist/types.d.ts:2:21 - error TS7016: Could not find a declaration file for module 'bn.js'. '.../ethereumjs-account/node_modules/bn.js/lib/bn.js' implicitly has an 'any' type.
  Try `npm install @types/bn.js` if it exists or add a new declaration (.d.ts) file containing `declare module 'bn.js';`

2 import BN = require('bn.js');
```